README.md

```diff
@@ -50,13 +50,187 @@ module "postgresql" {
 }
 ```
 
+```hcl
+
+module "opensearch_domain" {
+  source = "github.com/camunda/camunda-tf-eks-module/modules/opensearch"
+
+  domain_name     = "my-opensearch-domain"
+  subnet_ids      = module.eks_cluster.private_subnet_ids
+  security_group_ids = module.eks_cluster.security_group_ids
+  vpc_id          = module.eks_cluster.vpc_id
+  cidr_blocks      = concat(module.eks_cluster.private_vpc_cidr_blocks, module.eks_cluster.public_vpc_cidr_blocks)
+
+  instance_type   = "t3.small.search"
+  instance_count  = 4
```

Comment:
how come an instance count of 4?

Reply:
I don't think 2 will work as you have zone aware distribution with 3 zones and 3 subnets.
It will want the minimum to be 3.